### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by suryabhupa**
Added simple requirements.txt for easy pip installation of dependencies and added a note in the README.md.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+scipy==0.18.1
+torch==0.1
```
Comment (kentsommer) [inline requirements.txt]: Pytorch cannot currently be installed directly from pypi. I would suggest removing this and adding a note in the readme pointing the user to install the proper version (via their requirements for cuda etc) from here: http://pytorch.org/

**Review by kentsommer — CHANGES_REQUESTED** [review]
- [ ] Fix Pytorch install issue  (see comment)
- [ ] Make all pip installs >= (want to specify min not absolute)

**Comment by suryabhupa** [comment]
haha, I totally forgot to remove torch from requirements.txt; fixed it for now. Let me know if there any other changes you'd like to see.

**Comment by kentsommer** [comment]
@suryabhupa 
Thanks for the quick fix, and thanks for the contribution!

**Comment by suryabhupa** [comment]
Not at all -- thank you for this implementation! It's really wonderfully done; maybe I'll add on to the list of VIN implementations now too haha. :)

**Pull request merged**